src/main.cpp

```diff
@@ -40,14 +41,15 @@ const std::string help
       "molecule. "
       "Helps against combinatorial explosion if core is small or has high symmetry (default: false)\n"
       "  --assemblies <amount>\t\t\tNumber of starting assemblies (default: 10)\n"
-      "  --core <algorithm>\t\t\tAlgorithm to detect core structure (default: mcs, allowed: mcs, murcko)\n";
+      "  --core <algorithm>\t\t\tAlgorithm to detect core structure (default: mcs, allowed: mcs, murcko)\n"
+      "  --confsLog <path>\t\t\tOptional path to folder to store the generated conformers\n";
 
 std::optional<ProgrammOptions> parseArgs(int argc, char* argv[]) {
     ProgrammOptions parsed_options;
 
     opts::options_description desc("Allowed options");
     desc.add_options()("help,h", "print help message")(
-        "file,f", opts::value<std::string>(&parsed_options.input_file_path)->required(), "path to input file")(
+        "file,i", opts::value<std::string>(&parsed_options.input_file_path)->required(), "path to input file")(
```

Comment:
Würde file zu inputfile oder so umbenennen und in den helpers muss das auch noch angepasst werden oben.